src/main/kotlin/cacophony/semantic/ASTGeneration.kt

```diff
@@ -292,11 +292,48 @@ private fun generateASTInternal(
     return Empty(range = Pair(Location(0), Location(0)))
 }
 
+private fun wrapInFunction(originalAST: AST): AST {
+    val beforeStart = Location(-1)
+    val behindEnd = Location(originalAST.range.second.value + 1)
+    val program =
+        Definition.FunctionDeclaration(
+            Pair(beforeStart, behindEnd),
+            "<program>",
```

Comment:
The string `"<program>"` occurs a few times in the code. I'm wondering whether we should create `const val PROGRAM_FUNCTION_IDENTIFIER = "<program>"` or `const val MAIN_FUNCTION_IDENTIFIER = "<program>"`. However, for now this might be an overkill, so this is just a suggestion, the decision is yours.